### DISCUSSION TITLE
Matt Butcher - Fermyon

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#149] [Announcements]
## Background

- https://twitter.com/technosophos
- https://twitter.com/Helmpack
- https://twitter.com/brigadecore
- https://twitter.com/krustlet
- https://twitter.com/fermyontech
- https://www.fermyon.com

## Potential Topics

- Web Assembly

## Introduction

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey everyone! Our guest today Matt Butcher. If you work with Kubernetes there's a good chance you've also used one of Matt's tools. Welcome Matt, so glad to have you. Before we start, would you like to tell our listeners a bit more about yourself?

### Background?

- [x] You've worked on a lot of open source tooling. What have you worked on? 
- [x] Why does the field of distributed applications interest you?
- [x] You helped co-author an illustrated children's guide to Kubernetes. What inspired that?

### Spin

- [x] What is spin? What problem prompted you and your team to build it?
- [x] Why WASM in the cloud?
- [x] What type of application would benefit from WASM in the cloud?
- [x] What is wasmtime? How do it enable what spin does?
- [x] How do you use it?
- [x] How does spin/wasm make writing secure applications easier? (capabilities) 
- [x] In the docs you show you can build js/ts to wasm. Doesn't this present any benefits?
- [x] Can you extend spin?

### Ferymon

- [x] What is OCI?
- [x] What is ferymon? What features does it built on top of spin?
- [x] The ferymon platform is open source and self hostable. You also offer a cloud platform, what extra features does that platform provide?

### Future

- [x] What do you think the future of building and deploying applications looks like?

**Comment by hipstersmoothie** [comment]
https://bloop.ai
https://qr-code-styling.com

**Comment by just-be-dev** [comment]
- https://www.datarabbit.com/
- https://www.inkandswitch.com/cambria/

**Comment by technosophos** [comment]
https://github.com/AlDanial/cloc
https://remarkable.com/
https://thebrowser.company/
